torch/_torch_docs.py

```diff
@@ -612,6 +634,28 @@ def merge_dicts(*dicts):
     tensor([-0.6387,     nan, -0.4552,     nan])
 """.format(**common_args))
 
+add_docstr(torch.asinh,
+           r"""
+asinh(input, out=None) -> Tensor
+
+Returns a new tensor with arcsinh (inverse hyperbolic sine) of the elements of :attr:`input`.
```

Comment:
Same language comment as above.

Reply:
Done, should be resolved here: https://github.com/pytorch/pytorch/pull/38388/commits/234df817a34e6d3450b9274fb27c13aa4b80b278.